### ISSUE TITLE
BC Daggers, great item, mechanically useless in 99% of situations.

### ISSUE THREAD

**Issue by Inames** [torilmud/issues#142]
The daggers die off too quickly while you are out of combat, and mobs die quickly and often, the daggers usually die off before you can re-engage. especially if you kill a mob with your circle. These daggers are fun to play with, but so ridiculously situational... please make them live for at least 15-20 seconds out of combat to give us time to re-engage when mobs die. Maybe even 30 seconds cuz sometimes you kill off a bunch of low level mobs that die quick before you get to the bigger target, and killing 3 mobs before you get to one that's beefy the daggers never get to engage and they die off.

OR

Just give them a standard life span inside of combat or out, whatever their current in-combat duration is... like 2m or whatever...

But I think just slightly extending the out of combat duration is less game-breaking.

**Comment by shevarash** [comment]
I agree, they have always been a cool idea with a problematic execution.

I'm more inclined to ditch the mechanic of creating a literal mob to help you and instead add messaging to that effect with the same result. Slight less cool perhaps, but more reliable.

**Comment by Inames** [comment]
I absolutely abhor that idea... I assume you mean to turn them into a damage proccing weapon that just uses verbage to describe that the dagger floats out to attack the mob then come back to your hands... which would basically make my 9 daggers that I've collected worthless... In which case I'd rather keep them how they are currently.

**Comment by Inames** [comment]
I think the execution of the item is great, the way it's been designed is good, there's not a lot of glaring issues with it. Eilistraee did quite a bit of tweaking to resolve existing issues, and the crash bugs were resolved years ago. I just simply hate the fact that the daggers will die in the period between killing a mob and re-engaging another mob. Increasing the longevity of the pet will keep the cool factor, resolve the concerns I have, and make them viable to use in more scenarios.

**Comment by Inames** [comment]
3 years and counting.
